packages/node/src/sdk/init.ts

```diff
@@ -130,31 +127,27 @@ function _init(
   }
 
   if (!isCjs()) {
-    const [nodeMajor, nodeMinor] = process.versions.node.split('.').map(Number);
-
-    // Register hook was added in v20.6.0 and v18.19.0
-    if (nodeMajor >= 22 || (nodeMajor === 20 && nodeMinor >= 6) || (nodeMajor === 18 && nodeMinor >= 19)) {
-      // We need to work around using import.meta.url directly because jest complains about it.
-      const importMetaUrl =
-        typeof __IMPORT_META_URL_REPLACEMENT__ !== 'undefined' ? __IMPORT_META_URL_REPLACEMENT__ : undefined;
-
-      if (!GLOBAL_OBJ._sentryEsmLoaderHookRegistered && importMetaUrl) {
-        try {
-          // @ts-expect-error register is available in these versions
-          moduleModule.register('@opentelemetry/instrumentation/hook.mjs', importMetaUrl);
-          GLOBAL_OBJ._sentryEsmLoaderHookRegistered = true;
-        } catch (error) {
-          logger.warn('Failed to register ESM hook', error);
-        }
-      }
-    } else {
-      consoleSandbox(() => {
-        // eslint-disable-next-line no-console
-        console.warn(
-          '[Sentry] You are using Node.js in ESM mode ("import syntax"). The Sentry Node.js SDK is not compatible with ESM in Node.js versions before 18.19.0 or before 20.6.0. Please either build your application with CommonJS ("require() syntax"), or use version 7.x of the Sentry Node.js SDK.',
-        );
-      });
-    }
+    registerEsmModuleHooks([
```

Comment:
Hmm, will this prevent other instrumentation from working, if people add their own OTEL instrumentation?